lib/gollum/public/gollum/stylesheets/template.scss.erb

```diff
@@ -53,6 +53,9 @@ h6 {
 
     &:before {
       content: url('data:image/svg+xml;utf8,<%= rocticon_css(:link) %>');
+      @media (prefers-color-scheme: dark) {
+        filter: invert(95%) sepia(4%) saturate(753%) hue-rotate(178deg) brightness(88%) contrast(92%);
```

Comment:
For posterity:

Since we're including the octicon SVG as content, we can't style it as a normal element (for instance by setting the `fill` color for `svg path`). This is a trick to transform the color of the svg. I used [this](https://codepen.io/sosuke/pen/Pjoqqp) to calculate the filters needed to get the color we want (which is just the main text color in dark mode).